src/utils/helpers.js

```diff
@@ -3,14 +3,14 @@ export function shortenAddress(address) {
 }
 
 export function findETHValue(ethObj) {
-  const fulfillmentAmount = parseInt(ethObj.fulfillmentAmount, 10);
-  const tokenDecimals = parseInt(ethObj.tokenDecimals, 10);
+  const fulfillmentAmount = parseInt(ethObj.fulfillmentAmount);
+  const tokenDecimals = parseInt(ethObj.tokenDecimals);
   return fulfillmentAmount / Math.pow(10, tokenDecimals);
 }
 
 export function bytesToSize(bytes) {
   var sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB'];
-  if (bytes == 0) return '0 Byte';
+  if (bytes === 0) return '0 Byte';
```

Comment:
Is there ever a chance that the `bytes` from the state is not a number, and needs to be implicitly converted from a string?

Reply:
I think == 0 was intentional if the field was falsey. I don't remember though :). I should've left a comment if that were the case.

I think it can be returned as falsey or a number.

Reply:
Cool, if we're not sure then I'll add a comment disabling the check here for now 👍